purchase_sale_stock_inter_company/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-Today Odoo SA
+# Copyright 2016-2019 Chafique DELLI @ Akretion
+# Copyright 2018-2019 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Inter Company Module for Purchase to Sale Order",
```

Comment:
```suggestion
    "name": "Inter Company Module for Purchase to Sale Order",
```
I would change the name to avoid having two modules with the same name.